plugins/chunter-resources/src/components/ThreadComment.svelte

```diff
@@ -53,9 +55,31 @@
         action: chunter.actionImpl.SubscribeComment
       } as Action)
 
+  const pinnedQuery = createQuery()
+  let pinned = false
+
+  pinnedQuery.query(
```

Comment:
too many queries, let's move it to ChannelView and ThreadView, and pass  `export let pinned: boolean = false` to this component. Currently you one query for PinnedMessages, and query for each! message